packages/compartment-mapper/src/node-modules.js

```diff
@@ -444,6 +466,7 @@ const graphPackages = async (
   tags,
   mainPackageDescriptor,
   dev,
+  commonDependencies = {},
```

Comment:
We can default this higher up the call tree so the dependency isn’t optional, and thereby reduce throw-away allocations.

Reply:
this is the default in the lowest level before recursion, so it should only be called once